torch/nn/quantized/modules/linear.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from ...modules.module import Module
+from ...._jit_internal import weak_module
+
+
+
+@weak_module
+class Linear(Module):
+    r"""
+    A module that wraps the quantized fbgemm linear operator function
+    We adopt the same interface as `torch.nn.Linear`, please see https://pytorch.org/docs/stable/nn.html#torch.nn.Linear
+    for documentation.
+
+    Similar to `torch.nn.Linear`, attributes will be randomly initialized at
+        module creation time and will be overwritten later
+
+    Attributes:
+        _packed_weight: the weight that is first quantized and then packed using
+            fbgemm_linear_pack function
+        bias:   the quantized bias
+        output_scale: `scale` parameter of output Quantized Tensor
+        output_zero_point: `zero_point` parameter for output Quantized Tensor
+    """
+    def __init__(self, qweight, qbias, output_scale, output_zero_point):
+        super(Linear, self).__init__()
```

Comment:
@dzhulgakov should we pass in original mod here as well? it has all the original arguments like `in_features`, `out_features` and I think we will need them for debugging purposes?

Reply:
As we discussed, we probably want to allow to create quantized module by supplying weights directly. Thus it's better to keep __init__ like this and offer `from_float` if you want to convert the original module.

For in/out features - you don't need them - you have qweight.size(0), qweight.size(1) :-)